libvast/src/plugin.cpp

```diff
@@ -380,6 +380,38 @@ store_plugin::make_store(system::accountant_actor accountant,
                                                  std::move(path), name());
 }
 
+// -- loader plugin -------------------------------------------------------------
+
+auto stdin_loader_plugin::make_loader(options, const bool_operator*) const
+  -> caf::expected<input_loader> {
+  return []() -> detail::generator<chunk_ptr> {
+    auto input = std::string{};
+    auto c = char{};
+    while (std::cin.get(c)) {
```

Comment:
This should most likely do unbuffered reads via the underlying streambufs.

We have some utilities (IIRC `fdinbuf`) where you can also set the buffer size.